continuedev/continue
# fix: scroll issues w/ code blocks

## Description

Fixes some of the scroll issues w/ code blocks.

## Checklist


## Screenshots

If your updates include visual changes, please share screenshots below.

## Testing

- Generate enough responses to fill the chat sidebar 
- `cmd+L` to add some code 
- Confirm that the chat window scrolls with the text streaming
- `cmd+L` to add some code and scroll up as it generates
- Confirm that that window doesn't jump to keep scrolling
- Scroll to bottom and confirm that it resumes auto scrolling
